### DISCUSSION TITLE
Socket (low level interface) with a flask app

### DISCUSSION THREAD

**Discussion by ZohaibDurrani** [miguelgrinberg/Flask-SocketIO#1657] [Q&A]
Hello, I used socket in a flask app but it doesn't Working. Kindly review Question and Code on,
stackoverflow:  https://stackoverflow.com/questions/68641976/flask-app-using-socket-low-level-interface

**Comment by miguelgrinberg** [comment]
Your question has nothing to do with Socket.IO.
